superdesk/metadata/item.py

```diff
@@ -452,6 +452,31 @@
             }
         }
     },
+
+    # track references to other objects,
+    # based on associations but allows queries
+    'refs': {
```

Comment:
I think we might need a migration to fill items `refs` based on their existing associations.

Reply:
ansa won't need it as they only care about last 24h, but if anyone else would want that to work for all content ever then yes